ReSharper.FSharp/src/FSharp.Psi.Intentions/src/QuickFixes/UpdateMutabilityInSignatureFix.fs

```diff
@@ -0,0 +1,71 @@
+namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.Highlightings
+open JetBrains.ReSharper.Psi.ExtensionsAPI
+open JetBrains.ReSharper.Resources.Shell
+open JetBrains.ReSharper.Plugins.FSharp.Psi
+
+type UpdateMutabilityInSignatureFix(error: ValueNotContainedMutabilityAttributesDifferError) =
+    inherit FSharpQuickFixBase()
+
+    let tryFindBindingSignature (implementationDeclaredName: string) (declaredElement: IFSharpMember) =
+        let parentDeclarations = declaredElement.ContainingType.GetDeclarations()
```

Comment:
It should be possible to just do `declaredElement.GetDeclarations`, unless there's some issue. It would return other patterns declaring this element.

Reply:
> It doesn't seem to work when the return type of the signature is different.

Yes, for now it's considered to be two different symbols because of the signature mismatch. That's something I'll need to look into when there're more features like this one are added.